src/modules/portfolio/containers/markets.js

```diff
@@ -33,6 +37,7 @@ const mapDispatchToProps = dispatch => ({
   collectMarketCreatorFees: (marketId, callback) => dispatch(collectMarketCreatorFees(marketId, callback)),
   loadMarketsInfo: marketIds => dispatch(loadMarketsInfo(marketIds)),
   toggleFavorite: marketId => dispatch(toggleFavorite(marketId)),
+  updateModal: modal => dispatch(updateModal(modal)),
 })
 
```

Comment:
these changes are not needed if properties are only needed at market-properties level. market-properties component container can inject them.